src/main/java/frc/robot/subsystems/swervedrive/SwerveSubsystem.java

```diff
@@ -101,6 +104,13 @@ public SwerveSubsystem(File directory) {
         swerveDrive.setAngularVelocityCompensation(true, true, 0.1); //Correct for skew that gets worse as angular velocity increases. Start with a coefficient of 0.1.
         swerveDrive.setModuleEncoderAutoSynchronize(false, 1); // Enable if you want to resynchronize your absolute encoders and motor encoders periodically when they are not moving.
         //    swerveDrive.pushOffsetsToEncoders(); // Set the absolute encoder to be used over the internal encoder and push the offsets onto it. Throws warning if not possible
+     
+        sqorchestra = new Orchestra();
```

Comment:
I think this would be more readable with a `this.` so that it's more obvious that this is referring to the class variable not a local variable.